woocommerce/wc-admin
# Add BACS as default fallback payment gateway

Fixes (part of) #6855 

* Adds in BACS to default gateways.
* Adds in a registered BACS plugin to set not-settings fields.
* Does seem clean-up around components/prop usage.

### Screenshots
<img width="689" alt="Screen Shot 2021-05-25 at 2 54 04 PM" src="https://user-images.githubusercontent.com/10561050/119573690-9f3e7200-bd82-11eb-87a4-adf29b65a18a.png">

### Detailed test instructions:

1. Deactivate the remote tester plugin and delete any relevant transients.
2. Don't select CBD for an industry
2. Visit the task list.
3. Fill out the form and make sure you can complete the payment setup.
4. Visit the WC Settings to manually check if the details were properly saved.
